conf_files/pocs.yaml

```diff
@@ -37,8 +37,9 @@ mount:
         baudrate: 9600
     non_sidereal_available: True
 pointing:
-    threshold: 0.05
-    exptime: 30
+    auto_correct: True
```

Comment:
If `auto_correct: False`, what benefit would we get? I think that logging in that case should at least be INFO level.

Reply:
Now that I think about it, is `auto_correct: False` supposed to completely suppress the pointing state?

Reply:
It suppresses the pointing correction, so an initial 30sec image will be taken but the actual `calibrate_mount` command will never be called. We use a pointing image for the WCS tracking correction (and a few other things).